Sources/Testing/Testing.docc/OrganizingTests.md

```diff
@@ -132,29 +132,29 @@ availability at runtime, a test suite type (and any types that contain it) must
 _not_ be annotated with the `@available` attribute:
 
 ```swift
-@Suite struct FoodTruckTests { ... } // ✅ OK: type is always available
+@Suite struct FoodTruckTests { ... } // ✅ OK: The type is always available.
 
-@available(macOS 11.0, *) // ❌ ERROR: suite type must always be available
+@available(macOS 11.0, *) // ❌ ERROR: The suite type must always be available.
 @Suite struct CashRegisterTests { ... }
 
-@available(macOS 11.0, *) struct MenuItemTests { // ❌ ERROR: suite type's
+@available(macOS 11.0, *) struct MenuItemTests { // ❌ ERROR: The suite type's
                                                  // containing type must always
-                                                 // be available too
+                                                 // be available too.
   @Suite struct BurgerTests { ... }
 }
 ```
 
-The compiler will emit an error when presented with a test suite that does not
+The compiler emits an error when presented with a test suite that doesn't
 meet this requirement.
 
-### Classes must be final
+#### Classes must be final
 
-The testing library does not currently support inheritance between test suite
-types. If a class is used as a test suite type, it may inherit from another
+The testing library doesn't support inheritance between test suite
```

Comment:
We're actually going to fix this soonish since the constraint on non-final types doesn't exist with swift-syntax-600.

Reply:
It's already actually technically not true anymore. 🤔

Reply:
Can we resolve this for now and then revisit this section later if things change?

Reply:
We can leave this point for later.